packages/react/src/primitives/TextField/TextField.tsx

```diff
@@ -30,6 +30,7 @@ const TextFieldPrimitive = <Multiline extends boolean>(
     descriptiveText,
     errorMessage,
     hasError = false,
+    height, // @TODO: remove custom destructuring for 3.0 release
```

Comment:
For my understanding, wdym by custom destructuring? And how is it a breaking change?

Reply:
Removing width and height from this de-structure and instead using this logic for which style props go to which underlying element: https://github.com/aws-amplify/amplify-ui/pull/1373

Reply:
Ah gotcha, thanks!

Reply:
We're specifically pulling the width and height and out of the base style props and applying them to the Flex container in this change. We'll also be doing this for all base style props in the 3.0, which is a much bigger change we will leave in the 3.0 release and won't do here. Thus, I added a comment to remove line when we release 3.0 since it will be done already.